HW_6/js.js

```diff
@@ -0,0 +1,117 @@
+//////////////////////////////////////////// Task 1
+
+var basket = [];
+var products = [
+    {product: "The first item", price: 2, additionalPhotos: ["img/хаски1.jpg", "img/хаски2.jpg"]},
+    {product: "The second item", price: 3, additionalPhotos: ["img/белый 1.jpg", "img/белый 2.jpg"]},
+    {product: "The third item", price: 1, additionalPhotos: ["img/медведь 1.jpg", "img/медведь 2.jpg", "img/медведь 3.jpg"]},
+    {product: "The 4 item", price: 18, additionalPhotos: ["img/овчарка 1.jpg", "img/овчарка 2.jpg", "img/овчарка 3.jpg"]},
+    {product: "The 5 item", price: 15, additionalPhotos: ["img/щенок 1.jpg", "img/щенок 2.jpg", "img/щенок 3.jpg", "img/щенок 4.jpg"]}
+];
+var timerId;
+var modal = document.querySelector("#modal");
+var modalOverlay = document.querySelector("#modal-overlay");
+var closeButton = document.querySelector("#close-button");
+
+function displayCart(cartContentTotal) {
+    var $parent = document.getElementById('cart');
+    var $child = document.querySelector(".wrapCart p") ? document.querySelector(".wrapCart p") : document.createElement("p");
+    if (cartContentTotal.quantity == 0) {
```

Comment:
Лучше использовать всегда строгое сравнение. Чем строже вы пишете - тем меньше потенциальных ошибок
```suggestion
    if (cartContentTotal.quantity === 0) {
```